projects/ngneat/transloco-persist-lang/LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License
```

Comment:
do we need a license for each library? I think that one on the root is enough

Reply:
No, it's a separate package.